TKGame/Level Editor Content/LevelEditor.cs

```diff
@@ -360,10 +391,27 @@ public static Stage LoadStageDataFromJSON(string stageName)
 
             StageData levelData = JsonSerializer.Deserialize<StageData>(jsonString);
 
-            foreach (WallData wallData in levelData.walls)
+            foreach (BlockData blockData in levelData.blocks)
             {
-                Wall newWall = new Wall(wallData.X, wallData.Y, wallData.width, wallData.height, Color.White);
-                newStage.StageWalls.Add(newWall);
+                IBlock newBlock;
+                if (blockData.type == "Wall")
+                {
+                    newBlock = new Wall(blockData.X, blockData.Y, blockData.width, blockData.height, Color.White);
+                }
+                else if (blockData.type == "Spikes")
+                {
+                    newBlock = new Spikes(blockData.X, blockData.Y, blockData.width, blockData.height);
+                }
+                // TODO:
+                //else if (blockData.type == "Platform")
+                //{
+                //    newBlock = new Platform(blockData.X, blockData.Y, blockData.width, blockData.height, Color.White);
+                //}
+                else
+                {
+                    newBlock = new Wall(blockData.X, blockData.Y, blockData.width, blockData.height, Color.White);
+                }
+                newStage.StageBlocks.Add(newBlock);
```

Comment:
Just to be picky, couldn't you say:
if(blockData.type == "Spikes")
{
// spikes
}
else
{
// wall
}
Not important but just to make code more concise since those are the only two types atm